toolkit/components/uniffi-bindgen-gecko-js/fixtures/Arithmetic.jsm

```diff
@@ -1,6 +1,8 @@
 // This file was autogenerated by some hot garbage in the `uniffi` crate.
```

Comment:
as a meta-comment, I wonder if we should use a directory name like `__generated__` for generated stuff we check in? I started adding a couple of comments here before realizing :)